apps/script/HoC2023ScriptFiles/constants.py

```diff
@@ -8,78 +8,81 @@
 
 # We rename foreground/background effects and palettes as their model_descriptive_name
 # to better reflect the actual output of the effect or color.
+# 29 total backgrounds
 BACKGROUND_EFFECT_MODEL_DESCRIPTIVE_NAMES = [
-    'moving shapes',
+    'geometric shapes',
     'flower petals',
-    'pulse',
+    'round circle geometric',
     'clouds',
-    'solid colors',
+    'simple',
     'diamonds',
     'disco',
-    'firework',
-    'flowers',
+    'explosion celebration',
+    'flowers plants',
     'light grid',
     'stars',
-    'groovy shapes',
-    'swirl',
+    'groovy',
+    'swirl twist',
     'kaleidoscope',
-    'laser beam',
+    'electronic',
     'paint splatter',
-    'rainbow',
-    'dots',
-    'circles',
-    'snow fall',
-    'words',
-    'galaxy',
-    'sparkles',
-    'spiral',
-    'squares',
+    'rainbow stripes',
+    'dots geometric pattern',
+    'round shapes geometric',
+    'snow',
+    'words song music',
+    'galaxy space',
+    'bright',
+    'spiral twisty',
+    'grid geometric',
     'squiggles',
     'starburst',
     'twinkle',
-    'sound wave'
+    'music wave geometric'
 ]
 
+# 21 total palettes
 PALETTE_MODEL_DESCRIPTIVE_NAMES = [
```

Comment:
Something I maybe should already understand, apologies...are we keeping these string and the ones in `ai-inputs` in sync by hand? Or is one pulled from the other?

Reply:
The `modelDescriptiveNames` in `constants.py` are only for the outputs (background/foreground effects and palettes) and pulled in [here](https://github.com/code-dot-org/code-dot-org/blob/b01a976fca8ee0676946eee61a2bfc6ccec4b1bd/apps/script/HoC2023ScriptFiles/HoC2023AiGenerateWeights.py#L24-L26) in `HoC2023AiGenerateWeights.py`. 

The `modelDescriptiveNames` for emoji inputs are stored in `ai-inputs.json` and pulled in via `get_ai_emoji_inputs` function defined [here](https://github.com/code-dot-org/code-dot-org/blob/b01a976fca8ee0676946eee61a2bfc6ccec4b1bd/apps/script/HoC2023ScriptFiles/HoC2023AiHelperFunctions.py#L20) which is called in `HoC2023AiGenerateWeights.py`. 

I thought about placing the `modelDescriptionNames` for the emojis in this same `constants.py` file, but I think since this `ai-inputs.json` file is specifically for the AI modal only and it's nice for a reader to see the emoji, emoji id along with the model description name used, I kept the descriptive names here.

Reply:
Got it, thanks!